cico_run_tests.sh

```diff
@@ -0,0 +1,19 @@
+
+#!/bin/bash
+
+. cico_setup.sh
+
+TAG="test"
+TEST_IMAGE_NAME="$(make get-image-name):$TAG"
```

Comment:
@echo $(REGISTRY)/$(REPOSITORY)-tests:$(DEFAULT_TAG) add this in makefile and use that here. Otherwise in its current state, doesn’t it name the test image also as the main image?

Reply:
The names are same but tags are different. Simpler for us. We don't neet to maintain different versions of the test image. We just need one image. So, tagging as $IMAGE_NAME:test made sense to me.

Reply:
No it’s not simple the image in the local cache will be overwritten